packages/amplify_flutter/test/config/testdata/src/yarn.lock

```diff
@@ -0,0 +1,364 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
+amplify-frontend-flutter@^1.3.0:
```

Comment:
Why are we checking in a yarn.lock?

Reply:
Thought this was common practice in JS land?

Reply:
It is common practice; however, it can cause some headaches when all contributers aren't the same version of yarn or are using npm instead. Given the very limited scope of our usage I think it's ok. We just need to be very cognizant of changes to this file.